app/views/layouts/application.html.haml

```diff
@@ -13,5 +13,7 @@
         = yield
         = render 'layouts/footer'
 
+    %javascript_include_tag 'validate', 'data-turbolinks-track' => true
```

Comment:
You should just include this into the asset pipeline rather than a include tag here.